libkmod/libkmod-module.c

```diff
@@ -240,6 +240,10 @@ static int kmod_module_new(struct kmod_ctx *ctx, const char *key, const char *na
 	size_t keylen;
 	int err;
 
+	/* TODO: rework to create the hash within this function and remove the following
+	 * workaround */
+	(void) name; /* silence unused parameter warnings */
```

Comment:
humn... using (void) to silence warnings remembered me of a gcc bug I commented on a very long time ago.
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66425#c10

it was for unused return, but still brings bad memory for adopting it here